community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -180,7 +180,7 @@
     @Internal
     public static final Setting<String> cypher_runtime = setting(
             "unsupported.cypher.runtime",
-            options( "INTERPRETED", "COMPILED", DEFAULT ), DEFAULT );
+            options( "INTERPRETED", "COMPILED", "ENTERPRISE-INTERPRETED" ,  DEFAULT ), DEFAULT );
```

Comment:
I still think that this is a really bad name, and should not make it out to users. Can't we change it to the "turbo-runtime" until we have something better? We want to be able to move this stuff out of enterprise, and calling it "enterprise-interpreted" makes that really hard.